README.md

```diff
@@ -17,13 +17,14 @@ This means it's not possible to use anything in the [`std`](https://doc.rust-lan
 
 ## SDK version
 
-Currently supports SDK 28.2 ([flipperzero-firmware@0.84.1](https://github.com/flipperdevices/flipperzero-firmware/tree/0.84.1)).
+Currently supports SDK 30.1 ([flipperzero-firmware@0.85.2](https://github.com/flipperdevices/flipperzero-firmware/tree/0.85.2)).
 
 The crate major version number will be updated after a bump in [API version](https://github.com/flipperdevices/flipperzero-firmware/blob/release/firmware/targets/f7/api_symbols.csv) in the Flipper Zero firmware.
 
 | Crate version | API version |
 | ------------- | ----------- |
-| 0.10.x        | 28.2        |
+| 0.10.1        | 30.1        |
```

Comment:
This violates the documented version bumping policy above: we don't change the SDK API version in a SemVer-compatible release, because FAPs built with different API versions to the Flipper Zero firmware will not run. The next crate release with an SDK bump will be `0.11`.

Reply:
reverted